app/src/main/java/fr/free/nrw/commons/delete/DeleteTask.java

```diff
@@ -181,6 +186,10 @@ protected void onPostExecute(Boolean result) {
                 .setProgress(0,0,false)
                 .setOngoing(false)
                 .setPriority(PRIORITY_HIGH);
+        String urlForDelete=baseUrl+media.getFilename();
+        Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(urlForDelete));
```

Comment:
@vanshikaarora you can use CustomTabs instead of webview. I think there is a function handleWebUrl in ViewUtils.java for that.